app/client/views/confirmation/confirmation.html

```diff
@@ -24,6 +24,21 @@
                    required>
           </div>
 
+          <div class="field">
+            <label>Race/Ethnicity</label>
+            <select
+              name="ethnicity"
+              ng-model="user.profile.ethnicity">
```

Comment:
I guess since someone already signed up, it would be easier to keep it under profile, but ideally we should move this to user.confirmation and update the database and User.js

Reply:
We can do that later if you wanna get this changed ASAP

Reply:
hmm yeah lets do that